kola/harness.go

```diff
@@ -107,6 +108,18 @@ func filterTests(tests map[string]*register.Test, pattern, platform string) (map
 			continue
 		}
 
+		var minVersion *semver.Version
+		if t.MinVersion == "" {
```

Comment:
If `t.MinVersion` where a `semver.Version` instead of a `string`, this entire block could be removed.